extensions/arc/package.json

```diff
@@ -212,9 +212,99 @@
                           "resourceGroupVariableName": "AZDATA_NB_VAR_ARC_RESOURCE_GROUP"
+                        },
+                        {
+                          "type": "azure_locations",
+                          "label": "%arc.data.controller.arc.data.controller.location%",
+                          "defaultValue": "eastus",
+                          "required": true,
+                          "locationVariableName": "AZDATA_NB_VAR_ARC_DATA_CONTROLLER_LOCATION",
+                          "displayLocationVariableName": "AZDATA_NB_VAR_ARC_DATA_CONTROLLER_DISPLAY_LOCATION",
+                          "locations": [
+                            "australiaeast",
+                            "centralus",
+                            "eastus",
+                            "eastus2",
+                            "francecentral",
+                            "japaneast",
+                            "koreacentral",
+                            "northeurope",
+                            "southeastasia",
+                            "uksouth",
+                            "westeurope",
+                            "westus2"
+                          ]
                         }
                       ]
                     },
+                    {
+                      "title": "%arc.data.controller.connectivitymode%",
+                      "fields": [
+                        {
+                          "type": "readonly_text",
+                          "label": "%arc.data.controller.data.controller.connectivitymode.description%",
+                          "labelWidth": "600px"
+                        },
+                        {
+                          "type": "options",
+                          "label": "%arc.data.controller.connectivitymode%",
+                          "required": true,
+                          "variableName": "AZDATA_NB_VAR_ARC_DATA_CONTROLLER_CONNECTIVITY_MODE",
+                          "options": {
+                            "values": [
+                              {
+                                "name": "indirect",
+                                "displayName": "%arc.data.controller.indirect%"
+                              },
+                              {
+                                "name": "direct",
+                                "displayName": "%arc.data.controller.direct%"
+                              }
+                            ],
+                            "defaultValue": "%arc.data.controller.indirect%",
+                            "optionsType": "radio"
+                          }
+                        },
+                        {
+                          "label": "%arc.data.controller.spnclient%",
+                          "variableName": "AZDATA_NB_VAR_SPN_CLIENT_ID",
+                          "type": "text",
+                          "required": true,
+                          "defaultValue": "",
```

Comment:
for fields with no deafultValue, we should consider if there is a relevant 'placeholder' that we can place to inform the user on how to fill the value.

Reply:
I'll be following up with an info link and/or descriptions for these fields once I have something for those - this is to get the feature out for testing initially and then I'll do polishing like that next.